.circleci/e2e_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# NOTE: removing +x mode can reveal credentials in circleci logs
+set +x
+set -eo pipefail
+
+cp -a artifacts/*crx .
+cp -a artifacts/*xpi .
+npm run test-e2e-${BROWSER}
+mv screenshots screenshots-${BROWSER}
```

Comment:
yep, it works because they are persisted in workspace layers that are stacked before running `deploy-artifacts`
This per-browser naming avoided collisions on workspace layers attachments, so I just did the workaround in the fastest place for me, but yeah, it could be have done from `js` also!